### DISCUSSION TITLE
在数据入库之前获取Context的某个值

### DISCUSSION THREAD

**Discussion by 7kyun** [midwayjs/midway#2755] [Q&A]
是这么个需求
在入库之前，需要获取`Context`中的userId，用作表的create_by和update_by
请问有没有比较优雅的解决方案，TypeOrm的订阅器试了下好像拿不到Context的数据，然后像试试拦截器来对入参进行修改

所以这边问问各位大佬有没有遇到过这种问题，求个思路

**Comment by waitingsong** [comment]
限于 TypeOrm ？

**Comment by pionear-layen** [comment]
新版可以。
